docs/advanced-recipes/atom-creators.mdx

```diff
@@ -291,3 +291,76 @@ function EvenCounter() {
   return <>Count was set to an even number {evenCount} times.</>
 }
 ```
+
+## atomWithBroadcast
+
+> `atomWithBroadcast` creates an atom. The atom will be shared between
+> browser tabs and frames, similar to `atomWithStorage` but with the 
+> initialization limitation.
+
+This can be useful when you want the state to interact with each other without
+the use of localStorage and that by just using The Broadcast Channel API allows
+basic communication between browsing contexts (that is, windows, tabs, frames, 
+create a component or iframes) and workers on the same origin. According to the MDN documentation receiving a message in initialization is not supported in broadcast and if we want to support that we may need to add extra stuff to atomWithBroadcast (like local storage).
+
+```tsx
+import { atom } from "jotai";
+
+const channelCache = new Map<string, BroadcastChannel>();
```

Comment:
My apologies if my comment was confusing.
We don't need to deal with two `atomWithBroadcast` invocation with the same `key`.
```suggestion
```